lib/util.js

```diff
@@ -486,6 +494,16 @@ function formatMap(ctx, value, recurseTimes, visibleKeys, keys) {
   return output;
 }
 
+function formatIterator(ctx, value, recurseTimes, visibleKeys, keys) {
+  var output = [];
+  for (let o of value) {
```

Comment:
So I guess that means that we will have to use Debug.MakeMirror and fetch the values?